src/components/common/Bar.vue

```diff
@@ -2,8 +2,14 @@
   <div v-if="showMessage" :class="`bar ${barType}`">
     <p>
       <slot />
+      <span v-if="link" class="link" @click="goToLink(link)">{{ link }}</span>
```

Comment:
replace with `router-link`?

Reply:
I don't think it makes that much difference since you still need to do the window.location thing and for that you need a method, which renders the native `to` from `router-link` useless

Reply:
why do you need `window.location`?

Reply:
because if you do `<router-link to="https://lunie.io`>' and you click it, it will take you to `https://app.lunie.io/<currentNetworkId>/https://lunie.io`

Reply:
if it's always an http full url we can use an `a` tag..?